### DISCUSSION TITLE
Delete an Issue

### DISCUSSION THREAD

**Discussion by KipoDipo** [community/community#51108] [Other Feature Feedback, Questions, & Ideas]
There isn't a (visible) way to delete an issue from the mobile version.

**Comment by aquaaerobicsystem** [comment]
GitHub issue down right now.

![image](https://user-images.githubusercontent.com/46786408/227942482-98acbec1-72ab-4c43-9128-39be623dd1fa.png)

**Reply by KipoDipo** [reply]
I meant that there isn't such functionality

![image](https://user-images.githubusercontent.com/101924814/227943029-637d81e0-67ef-4206-bc9e-eb7c47988a39.png)

**Reply by KipoDipo** [reply]
I meant that there isn't such functionality

![image](https://user-images.githubusercontent.com/101924814/227943029-637d81e0-67ef-4206-bc9e-eb7c47988a39.png)

**Comment by aquaaerobicsystem** [comment]
Yes, you can do this on mobile. 
- Go to Repository then Settings > Danger Zone (at bottom of settings) 
- Tap "Delete this repository"
![image](https://user-images.githubusercontent.com/46786408/227944903-1f28778c-7cdb-4cf3-9e71-62708e1f22ad.png)

**Reply by KipoDipo** [reply]
I'm talking about deleting an Issue, not a Repository
